src/test/java/be/seeseemelk/mockbukkit/ServerMockTest.java

```diff
@@ -67,6 +66,21 @@ void class_NumberOfPlayers_Zero()
 		assertEquals(0, server.getOnlinePlayers().size());
 	}
 
+	@Test
+	void createWorld_WorldCreator() {
+		WorldCreator worldCreator = new WorldCreator("test")
+				.seed(12345)
+				.type(WorldType.FLAT)
+				.environment(World.Environment.NORMAL);
+		server.createWorld(worldCreator);
+
+		assertEquals(1, server.getWorlds().size());
+		assertEquals("test", server.getWorlds().get(0).getName());
+		assertEquals(12345, server.getWorlds().get(0).getSeed());
+		assertEquals(WorldType.FLAT, server.getWorlds().get(0).getWorldType());
+		assertEquals(World.Environment.NORMAL, server.getWorlds().get(0).getEnvironment());
```

Comment:
`Server#createWorld` returns the created World.
It would be much more sensible to store it in a var and use that for your assertions instead of `server.getWorlds().get(0)`

Reply:
Thats true, good point